jetstream/experimenter.py

```diff
@@ -61,6 +61,7 @@ class Experiment:
     is_high_population: bool
     app_name: str
     app_id: str
+    outcomes: List[str] = []  # todo: TBD in Experimenter
```

Comment:
This part needs to be modified based on how outcomes are exposed via the Experimenter API

Reply:
```suggestion
    outcomes: List[str] = attr.Factory(list)  # todo: TBD in Experimenter
```
can't imagine this mattering but if you'll indulge my superstition